core/src/main/java/io/confluent/kafka/schemaregistry/storage/SchemaRegistry.java

```diff
@@ -48,7 +48,7 @@ default Schema getByVersion(String subject, int version, boolean returnDeletedSc
   Schema get(String subject, int version, boolean returnDeletedSchema)
       throws SchemaRegistryException;
 
-  SchemaString get(int id) throws SchemaRegistryException;
```

Comment:
Do we need to still keep `SchemaString get(int id) throws SchemaRegistryException;`  and mark it deprecated?

Reply:
Good question.  We'll leave this API as is, it will use the default context to get the schema by ID.

Reply:
Actually this is on the server side, so not really used that often.  On the client side we will keep this API, which will use the default context.